osu.Game.Rulesets.Mania/Skinning/Argon/ManiaArgonSkinTransformer.cs

```diff
@@ -26,6 +28,34 @@ public ManiaArgonSkinTransformer(ISkin skin, IBeatmap beatmap)
         {
             switch (lookup)
             {
+                case SkinComponentsContainerLookup containerLookup:
+                    if (containerLookup.Target != SkinComponentsContainerLookup.TargetArea.MainHUDComponents)
+                        return base.GetDrawableComponent(lookup);
+
+                    // Only handle per ruleset defaults here.
+                    if (containerLookup.Ruleset == null)
+                        return base.GetDrawableComponent(lookup);
+
+                    // Skin has configuration.
+                    if (base.GetDrawableComponent(lookup) is UserConfiguredLayoutContainer d)
+                        return d;
```

Comment:
Both this and `ManiaLegacySkinTransformer` haven't been updated in line with d2eb6ccb8caa

Reply:
Have updated these.